crates/oxc_linter/src/config/env.rs

```diff
@@ -70,4 +75,17 @@ mod test {
         assert_eq!(env.iter().count(), 1);
         assert!(env.contains("builtin"));
     }
+
+    #[test]
+    fn test_override_envs() {
```

Comment:
might be good to add some more tests around this, though probably not critical since we have the end-to-end tests via the oxlintrc files. was thinking we could add a test for checking that you can: 1) replace the value (true -> false), 2) add a key (empty -> 1 key)

Reply:
I implemented the test in another place: https://github.com/oxc-project/oxc/pull/8915/files#diff-4803bbea196e86eff70b44ffa5fef5e38573bfeeaf90cb578cba00b20e2694acR302-R405
This fitted more because the overrides plugin/rules tests are here too